spec/parsers/pdf_parser_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
```

Comment:
This file could really use a spec for the unhappy path as well, since that would have caught several issues posted above.